spec/javascript/components/supplemental_application/__snapshots__/SupplementalApplicationPage.test.js.snap

```diff
@@ -1128,70 +1128,33 @@ Array [
             <div
               className="button-pager_row align-buttons-left primary inset-wide"
             >
-              <button
-                aria-expanded="false"
-                className="button dropdown-button has-icon--right text-align-left small is-approved small has-status-width"
-                disabled={false}
-                href="#"
-              >
-                <span
-                  aria-hidden="true"
-                  className="ui-icon ui-small"
-                >
-                  <svg>
-                    <use
-                      xlinkHref="#i-arrow-down"
-                    />
-                  </svg>
-                </span>
-                Approved
-              </button>
-              <ul
-                aria-activedescendant=""
-                aria-hidden="true"
-                className="dropdown-menu"
-                role="listbox"
-                style={
-                  Object {
-                    "display": "none",
-                  }
-                }
-                tabIndex={0}
+              <div
+                className="dropdown dropdown-inline"
+                onClick={[Function]}
+                style={undefined}
               >
-                <li
-                  aria-selected="false"
-                  className="dropdown-menu_item"
-                  role="option"
-                >
-                  <a
-                    href="/some/valid/uri"
-                  >
-                    This is a link
```

Comment:
I see what you are saying. The reason why the links are not showing is because we are using the `Dropdown` component. The previous code was just a "placeholder".  The way the Dropdown component works is that it renders the menu in the DOM only if it's expanded. Since we initially load the page, the dropdown is not expanded. Options are 1) changing the dropdown to render the menu using  css prop `display`, so we always render the menu in the DOM 2) leave it as it is and simulate a click in the dropdown in the test to render the menu. @callaghanc any thoughts?

Reply:
@fddayan I prefer 2), leave it as it is and simulate a click in the dropdown in the test to render the menu.

Reply:
has this been resolved yet?

Reply:
@akegan The e2e test addresses this by using the dropdown in its testing. Do you think we should add additional testing around this?

Reply:
@akegan check out the recently added unit test that tests the rendering of the status dropdown button and status dropdown menu on the supp app page